webapp/.eslintrc.json

```diff
@@ -1,5 +1,8 @@
 {
-  "extends": "eslint:recommended",
+  "extends": [
```

Comment:
are these changes to this file intended? if so, are they needed as part of this PR?

Reply:
This specific change was not intended. But the change below at line 39 is to remove the warning (i.e - Warning: React version not specified in eslint-plugin-react settings. See https://github.com/yannickcr/eslint-plugin-react#configuration .
) while running `make check-style` command.

Reply:
ok